KK_Pregnancy/PregnancyGameController.cs

```diff
@@ -52,9 +52,12 @@ protected override void OnEndH(HSceneProc proc, bool freeH)
                 var controller = heroine.chaCtrl.GetComponent<PregnancyCharaController>();
                 if (controller == null) throw new ArgumentNullException(nameof(controller));
 
+                //Allow pregnancy if enabled, or overridden, and is not currently pregnant
                 if (!controller.Data.GameplayEnabled || controller.Data.IsPregnant) return;
 
-                var winThreshold = Mathf.RoundToInt(controller.Data.Fertility * 100);
+                var fertility = (PregnancyPlugin.FertilityOverride.Value > controller.Data.Fertility) ? PregnancyPlugin.FertilityOverride.Value : controller.Data.Fertility;
```

Comment:
This will not allow characters to ever have less than 0.3 fertility since the setting's lowest value is 0.3 and it can't be disabled.

Reply:
The setting description makes it sound like the value will be forced to the exact value the setting gives you, so maybe change the wording of the setting and add a 0 option; or change the setting to use an enum that has an "Off" option alongside"OverrideTo50" etc, then have a switch here that hard sets the fertility value.